public/components/experiment_create/configuration/form/user_behavior_form.tsx

```diff
@@ -55,7 +55,7 @@ export const UserBehaviorForm = ({
             />
           </EuiFlexItem>
           <EuiFlexItem grow={1}>
-            <EuiFormRow label="K Value">
+            <EuiFormRow label="K Value" helpText="The depth in the result set to include.">
```

Comment:
In case we keep the file, same suggestion as above:
"The number of documents to include from the result list."